guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md

```diff
@@ -160,10 +158,9 @@ The example defines the following test steps and their sequence:
 
 ## Test step class  {#test-step}
 
-All test steps are located in `<module>/Test/TestStep` as PHP classes in the following format:
+All test steps are located in `<module>/Test/TestStep` as PHP classes with the following format:
```

Comment:
Could be:
"The  `<module>/Test/TestStep` directory contains each test step as a PHP class with the following format:"

Reply:
Thanks.